components/brave_sync/sync_devices.cc

```diff
@@ -163,17 +158,14 @@ void SyncDevices::Merge(const SyncDevice& device,
       break;
     }
     case jslib_const::kActionUpdate: {
-      //DCHECK(existing_device != nullptr);
       DCHECK(existing_it != std::end(devices_));
-      //*existing_device = device;
       *existing_it = device;
       *actually_merged = true;
       break;
     }
     case jslib_const::kActionDelete: {
-      //DCHECK(existing_device != nullptr);
-      DCHECK(existing_it != std::end(devices_));
```

Comment:
DCHECK should only be triggered in debug build and what QA used for test is release build.
This doesn't solve the issue.

Reply:
Thanks for review. So the issue I address here https://github.com/brave/browser-android-tabs/issues/933 is fixed by another PR completely, will close issue. For this PR will create a new issue.